stacks_queues/stack.py

```diff
@@ -12,27 +13,33 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_last(element)        
```

Comment:
The LinkedList implementation provided is doubly linked with a tail pointer, which makes adding at the end efficient. If this were singly-linked we would want to use operations at the head instead.